internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -64,35 +66,32 @@ type FBCRegistryPod struct { //nolint:maligned
 	// FBCContent represents the contents of the FBC file (string YAML).
 	FBCContent string
 
-	// FBCDir is the name of the FBC directory name where the FBC resides in.
-	FBCDir string
-
-	// FBCFile represents the FBC filename that has all the contents to be served through the registry pod.
-	FBCFile string
+	// FBCIndexRootDir is the FBC directory that exists under root of an FBC container image.
+	// This directory has the File-Based Catalog representation of a catalog index.
+	FBCIndexRootDir string
```

Comment:
Do we need an additional variable if this value is always going to be `/configs`? It isn't customizable in any way, right?

Reply:
Answered in [comment](https://github.com/operator-framework/operator-sdk/pull/5868#discussion_r896106428rl)